### ISSUE TITLE
Seperating COVID response from treatment effect

### ISSUE THREAD

**Issue by marianoteehan** [recurve-methods/comparison_groups#7]
If you use impact due to COVID as a stratification parameter, and your treatment was active during the onset of COVID, how do you know you're getting COVID impact and not treatment impact during stratification?

Per Steve Schmidt's question

**Comment by steevschmidt** [comment]
Thanks for posting this Mariano. Clarifying my question/concern --

If treated customers are assigned to stratification bins based on "COVID-like" changes in energy use during the COVID period (e.g. "fractional change in usage"), and they were undergoing program treatment at the same time, it seems to me energy changes due to the program might cause them to be put in the wrong bin, and get an incorrect adjustment.

Using the effect of COVID to stratify program participants seems problematic, at least for residential behavioral programs where energy profile changes can vary widely both from the program treatment(s) and from COVID impacts.

To avoid this, bins could be created based only on pre-COVID characteristics.

**Comment by mcgeeyoung** [comment]
That's right Steve. You couldn't use COVID as a parameter for a program that was in-flight at the onset of COVID.

**Comment by steevschmidt** [comment]
Got it, thanks for confirming.
